playground/nextjs-app-router/components/OnchainProviders.tsx

```diff
@@ -35,6 +35,7 @@ function OnchainProviders({ children }: { children: ReactNode }) {
         <OnchainKitProvider
           apiKey={ENVIRONMENT_VARIABLES[ENVIRONMENT.API_KEY]}
           chain={base}
+          projectId="cf6a4eee-9206-42bc-acf5-698d3f4f3681"
```

Comment:
can you use an envar instead?
I've added this as `NEXT_PUBLIC_PROJECT_ID` - please add to `ENVIRONMENT_VARIABLES`

Reply:
done